docs/manual-EN/2.query-language/4.statement-syntax/2.data-query-and-manipulation-statements/fetch-syntax.md

```diff
@@ -2,14 +2,16 @@
 
 The `FETCH` syntax is used to get vertex/edge's properties.
 
-## Fetch Vertex property
+## Fetch Vertex Property
 
 Use `FETCH PROP ON` to return a (list of) vertex's properties. Currently, you can get multiple vertices' properties with the same tag in one statement. You can use `FETCH` together with [pipe](../../3.language-structure/pipe-syntax.md) and [user defined variables](../../3.language-structure/user-defined-variables.md).
 
 ```ngql
 FETCH PROP ON {<tag_name_list> | *} <vertex_id_list> [YIELD [DISTINCT] <return_list>]
 ```
 
+Fetch `<tag_name_list>` is only available for version 1.0.1 or later version. If your **Nebula Graph** is earlier than 1.0.1, you can only fetch one type of vertices in a single query.
```

Comment:
"Fetch `<tag_name_list>`" ==> "The <tag_name_list> parameter can work with the `FETECH` statement only in Nebula Graph 1.0.1 or later version"?

If your Nebula Graph version is earlier than 1.0.1?